mindmeld/constants.py

```diff
@@ -325,5 +330,68 @@
 UNICODE_NON_LATIN_CATEGORY = "Lo"
 UNICODE_SPACE_CATEGORY = "Zs"
 
-
 SYSTEM_ENTITY_PREFIX = "sys_"
+
+# TEXT PREPRATION CONSTANTS
+EXCEPTION_CHARS = "\@\[\]\|\{\}'"
+
+DEFAULT_REGEX_NORM_RULES = [
```

Comment:
Thank you for this suggestion Travis! Went ahead and made each Regex rule its own class following this suggestion. Now compiling regex patterns and have moved the rules to the normalizers.py module as suggested.

Reply:
Switched back to the following to avoid inline comments to ignore style errors from pylint and my IDE.

```
class RegexNormalizerRule(Normalizer):

    def __init__(self, pattern: str, replacement: str):
        """Creates a RegexNormalizerRule instance."""
        self.pattern = pattern
        self.replacement = replacement
        self._expr = re.compile(self.pattern)

    def normalize(self, s):
        return self._expr.sub(self.replacement, s)
```

Here is an example of this being used:

`"SeparateAposS": RegexNormalizerRule(pattern=r"(?<=[^\s])'[sS]", replacement=" 's"),
`